jenkinsci/tuleap-git-branch-source-plugin
# User fork traits is no longer available

Part of  [story #18320](https://tuleap.net/plugins/tracker/?aid=18320) native support of pull requests in tuleap branch source jenkins
plugin

Now the user forks from a Tuleap project are always skipped and never
built by Jenkins.

How to test:
 - In your Tuleap project, make sure to have at least 1 fork
 - Create a new Jenkins project
=> In the configuration page, the `Personal forks` option is not
available anymore.
=> In the Jenkins log, you should see a message which said the fork
repository is ignored.
